skills/deploy/SKILL.md

```
@@ -33,33 +34,98 @@ free -h
+gpu_memory_utilization =
+  safe_vllm_bytes / cuda_total_bytes
+```
+
+`--gpu-memory-utilization` is a fraction of **total CUDA-visible memory**, not a
+fraction of memory left after speech loads. Round the result down, start
+conservatively, and never exceed `0.90`. Normalize all measurements to bytes
+before calculating. If any input is unavailable or the
+result leaves no clear headroom, do not guess: use cloud services or stop and
+ask the user.
+
+4. Compare the safe budget with the fixed Lightning recipe selected in
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)